src/client/src/routes/OrderTicketRoute/VoiceInput.tsx

```diff
@@ -80,7 +80,10 @@ export class VoiceInput extends React.PureComponent<Props, State> {
     sessionInstance: null,
   }
 
-  static getDerivedStateFromProps({ context, source, requestSession }: Props, state: State): Partial<State> {
+  static getDerivedStateFromProps(
+    { context, source, requestSession }: Props,
+    state: State,
+  ): Partial<State> {
     let next: any = null
     const { analyser, blob, destination }: any = state
```

Comment:
Not your PR, but why does this destructuring have an `any` type 🤔

Reply:
Added the correct type